src/Language/PureScript/Scheme/CodeGen/Transpiler.hs

```diff
@@ -403,14 +422,14 @@ moduleToLibrary (Module _sourceSpan _comments moduleName _path
             -- other possible replacements for the binder associated to the
             -- NamedBinder.
             go value (NamedBinder _ann ident binder) =
-              (:) (runIdent ident, value)
+              (:) (identToJs ident, value)
                   (go value binder)
 
     ----------------------------------------------------------------------------
 
     letToScheme :: [Bind Ann] -> Expr Ann -> SExpr
     letToScheme binds body
-      = let_ (concatMap bindToScheme binds) (exprToScheme body)
+      = letRecS (concatMap bindToScheme binds) (exprToScheme body)
```

Comment:
Let's make a ticket to investigate performance optimizations of using `let` vs `letrec` vs `let*` vs `letrec*`

Reply:
I see that we're synchronizing, this is now tracked in #60 :slightly_smiling_face:

Reply:
#60 here it goes